lib/literature/uploaders.ex

```diff
@@ -12,7 +12,7 @@ defmodule Literature.Uploaders do
   alias Literature.Config
 
   @extension_whitelist ~w(.jpg .jpeg .png)
-  @versions ~w(original jpg webp avif)a
```

Comment:
what happened to avif?

Reply:
Seems .avif works only for .png images not for .jpg

Reply:
could you let me know steps to reproduce error? as locally this works for me

```
➜  Downloads convert hurdlesa.jpeg a.avif
➜  Downloads identify a.avif
a.avif AVIF 1880x1253 1880x1253+0+0 8-bit sRGB 36520B 0.000u 0:00.002
```

Reply:
I upload the image with the .jpg extension